src/Sylius/Bundle/UiBundle/Block/BlockEventListener.php

```diff
@@ -23,6 +23,11 @@ final class BlockEventListener
 
     public function __construct(string $template)
     {
+        @trigger_error(
+            sprintf('Using "%s" to add blocks to the templates is deprecated since Sylius 1.7. Use "sylius_ui" configuration instead.', self::class),
+            \E_USER_DEPRECATED
```

Comment:
I would mention here explicitly when this is going to be removed from codebase. I assume 2.0, right?